int8 codeYaw 
int8 codeThrottle
int8 codePitch
int8 codeRoll
int8 codeArmGripOn
int8 codeArmGripOff
int8 codeStabilizeMode
int8 codeRemoteArmed
